gst/nnstreamer/include/nnstreamer_plugin_api_trainer.h

```diff
@@ -140,6 +176,10 @@ nnstreamer_trainer_probe (GstTensorTrainerFramework * ttsp);
 extern int
 nnstreamer_trainer_exit (GstTensorTrainerFramework * ttsp);
 
+extern void
+nnstreamer_trainer_notify_event (GstTensorTrainerEventNotifier * notifier,
+    GstTensorTrainerEventType type);
```

Comment:
In case we need to add some parameters for event in the future, let's add a void pointer after ```type```:

```
... type, void * data);```
```

Reply:
okay, thank you. I will add '''void * data'''.